draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -46,8 +46,8 @@
 
       <abstract>
         <t> This document describes a method to transport Internet Key Exchange
-        Protocol (IKE) and IPsec packets over a TCP connection for traversing
-        network middleboxes that may block IKE negotiation over UDP.  This
+        Protocol (IKE) and IPsec packets over a TCP connection in case routers or network middleboxes 
+        do not handle UDP, so that IKE negotiation over UDP would fail.  This
```

Comment:
I find this a bit harder to understand — it's not about boxes that don't handle UDP in general, it's about ones that block IKE over UDP specifically.

Since this is text from the original, I'd prefer to leave as is.

```suggestion
        Protocol (IKE) and IPsec packets over a TCP connection for traversing
        network middleboxes that may block IKE negotiation over UDP.  This
```

Reply:
It was Paul Wouters' suggestion to make text more neutral. Original text he thinks sounds like we design a circumvention protocol to bypass filtering policies which IETF he think is not mandated to design.

Reply:
Sure, I can reply via email, but I’d prefer not to change this now.